Scripts/Runtime/Core/Steps/CallMethodAnimationStep.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace BrunoMikoski.AnimationSequencer
+{
+    public class CallMethodAnimationStep : AnimationStepBase
+    {
+        [SerializeField]
+        private UnityEvent methodToCall;
+        
+        public override string DisplayName => "Method Call Step";
+        
+        public override float Duration { get; }
+
+        public override bool CanBePlayed() => true;
+        
+        public override string GetDisplayNameForEditor(int index)
+        {
+            string targetName = "NULL";
+            if (methodToCall != null)
+                targetName = methodToCall.GetPersistentTarget(0).name;
+
+            string persistentTargetNames = String.Empty;
+            for (int i = 0; i < methodToCall.GetPersistentEventCount(); i++)
+            {
+                persistentTargetNames = $"{String.Join(", ", methodToCall.GetPersistentTarget(i).name).Truncate(45)}";
+            }
+            
+            return $"{index}. {DisplayName}: {persistentTargetNames}";
+        }
+
+        public override void Complete()
+        {
+            methodToCall?.Invoke();
+        }
+    }
+}
```

Comment:
missing new line at the end of file.

Reply:
Here: 4b638bd3c6d2d7440fc617cc20949c07c107d01e